MOBILE_CLIENT/ANDROID/_HIGH/APK_BIOMETRIC_AUTHENTICATION_BYPASS/meta.json

```diff
@@ -12,6 +12,10 @@
   "categories": {
     "OWASP_MASVS_L2": [
       "MSTG_AUTH_8"
+    ],
```

Comment:
You first need to add the rules to RE. Add them with a key and mention the key here.